.github/workflows/dockerimage.yml

```diff
@@ -58,5 +58,28 @@ jobs:
             dockerfile: docker/web/Dockerfile
             pull_image_and_stages: false
 
+  build_and_publish_redis_web:
+      if: github.repository == 'cBioPortal/cbioportal'
+      runs-on: ubuntu-latest
+      steps:
+        - name: 'Checkout git repo'
+          uses: actions/checkout@v1
+        - name: Extract branch or tag name
+          # For the web docker image we don't publish it as latest
+          # just extract branch/tag name and strip v prefix
+          run: echo "##[set-output name=image_tag_names;]$(echo ${GITHUB_REF##*/} | sed 's/^v//g')"
+          id: extract_tags
+        - name: 'Docker build with cache'
+          uses: whoan/docker-build-with-cache-action@v5
+          with:
+            username: "${{ secrets.DOCKER_USERNAME }}"
+            password: "${{ secrets.DOCKER_PASSWORD }}"
+            image_name: cbioportal/cbioportal
+            image_tag: ${{ steps.extract_tags.outputs.image_tag_names }}-redis-web-shenandoah
+            context: .
+            build_extra_args: '{"--build-arg": "MAVEN_OPTS=-DskipTests -Dpersistence.cache_type=redis"}'
```

Comment:
this is the one new line compared to `build_and_publish_web` (there's prolly a better way in github actions to avoid duplication)

Reply:
this argument is new in `whoan/docker-build-with-cache-action@v5`, so had to upgrade that as well